emanifest-py/pyproject.toml

```diff
@@ -1,18 +1,18 @@
 [project]
 name = "emanifest"
-version = "2.0.7"
+version = "3.0.0"
```

Comment:
This will need to be 3.0.1

Reply:
Why do we need to skip 3.0.0? I know something happened when a workflow was accidentally initiated but what's the problem?

Reply:
I yanked the release on PyPI and that means the number is reserved but unusable. The files are still there on PyPI and the version number is taken, so we can't re-use it.

Reply:
It should be deleted then, not yanked. Besides, nobody pinned the version 3.0.0 to their requirements.txt while it was up for the few minutes. I'd rather not have someone add `emanifest==3.0.0` to their requirements.txt file and it not work because PyPI version 3.0.0 is really a 2.X release.